### PR TITLE
Fix typo duplication should be duplicated

### PR DESCRIPTION
**Pull request by coliff**
**Fixes**: #

- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

#### Short description of what this resolves:

Fix typo duplication should be duplicated

## -

-

👍 Thank you!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -97,14 +97,14 @@ <h3>Standard</h3>
...
                     <li><label class="checkbox" title="Tag must be paired."><input type="checkbox" id="tag-pair"> tag-pair</label></li>
                     <li><label class="checkbox" title="The empty tag must closed by self."><input type="checkbox" id="tag-self-close"> tag-self-close</label></li>
                     <li><label class="checkbox" title="Special characters must be escaped."><input type="checkbox" id="spec-char-escape"> spec-char-escape</label></li>
                     <li><label class="checkbox" title="Id must be unique."><input type="checkbox" id="id-unique"> id-unique</label></li>
                     <li><label class="checkbox" title="Src of img(script,link) must set value."><input type="checkbox" id="src-not-empty"> src-not-empty</label></li>
-					<li><label class="checkbox" title="&lt;title&gt; must be present in &lt;head&gt; tag.."><input type="checkbox" id="title-require"> title-require</label></li>
+		    <li><label class="checkbox" title="&lt;title&gt; must be present in &lt;head&gt; tag.."><input type="checkbox" id="title-require"> title-require</label></li>
```
Comment (waldyrious) [inline index.html]: Why this change?

File: index.html [hunk 2]
```diff
@@ -97,14 +97,14 @@ <h3>Standard</h3>
                     <li><label class="checkbox" title="Attribute name must be lowercase."><input type="checkbox" id="attr-lowercase"> attr-lowercase</label></li>
                     <li><label class="checkbox" title="Attribute value must closed by double quotes."><input type="checkbox" id="attr-value-double-quotes"> attr-value-double-quotes</label></li>
                     <li><label class="checkbox" title="Attribute must set value."><input type="checkbox" id="attr-value-not-empty"> attr-value-not-empty</label></li>
-                    <li><label class="checkbox" title="Attribute name can not been duplication."><input type="checkbox" id="attr-no-duplication"> attr-no-duplication</label></li>
+                    <li><label class="checkbox" title="Attribute name can not been duplicated."><input type="checkbox" id="attr-no-duplication"> attr-no-duplication</label></li>
```
Comment (waldyrious) [inline index.html]: This should say "be duplicated" instead of "been duplicated".

But perhaps a different wording would work even better:

```suggestion
                    <li><label class="checkbox" title="Attribute name can not occur more than once."><input type="checkbox" id="attr-no-duplication"> attr-no-duplication</label></li>
```

(That's because "duplicate" implies two copies, while this violation is true of any number of replicas).

Comment (coliff) [reply index.html]: 'can not occur more than once' - great suggestion - that's better.

File: index.html (hunk 1 above)
Comment (coliff) [reply index.html]: I was trying to make them line up, using the GitHub web interface so the tabs/spaces aren't quite aligned :-\

Comment (waldyrious) [reply index.html]: try copy-pasting the whitespace from the line above instead, that might work better.
